src/main/java/org/symphonyoss/symphony/messageml/bi/BiContext.java

```diff
@@ -60,6 +75,42 @@ public void updateItem(String itemName, String attributeName) {
     }
   }
 
+  /**
+   * Used for simple messageML elements (like Paragraphs, Links, Headers) where we only want to keep the count of attributes found.
+   * It checks if the context already has an item for the element and if that's the case it will increase the count of
+   * the specific attribute. If context does not have the item it will create a new one with default values.
+   *
+   * @param itemName      name of the element to be checked
+   */
+  public void updateItem(String itemName) {
+    String attributeName = "count";
+    Optional<BiItem> optionalBiItem = getItemWithName(itemName);
+    if (optionalBiItem.isPresent()) {
+      optionalBiItem.get().increaseAttributeCount(attributeName);
+    } else {
+      addItem(new BiItem(itemName, attributeName));
+    }
+  }
+
+  /**
+   * Used for simple messageML elements (like Paragraphs, Links, Headers) where we only want to keep the count of attributes found.
```

Comment:
looks like the doc was copy pasted here

Reply:
Fixed in this commit https://github.com/symphonyoss/messageml-utils/pull/266/commits/bedab46386b8a9405936838f80189140567d89d4